processor/modifyattributes/testdata/config.yaml

```diff
@@ -0,0 +1,65 @@
+receivers:
+  modify-attributes:
+    # The following is an example of renaming the
+    span-rename:
+      keys: ["db.svc", "operation"]
```

Comment:
If I understand correctly the key keys in the span should match the keys specified here for `span-rename` to have effect. How about we use `match-keys` instead of `keys` here to make it more apparent?